content/marketing/component-based.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Component-Based
```

Comment:
I considered putting the title inside the content instead but I think putting it in the frontmatter makes more sense so we leave the formatting of the title outside this file.

Reply:
Sounds reasonable to me.